libs/pxt-common/pxt-helpers.ts

```diff
@@ -535,7 +535,7 @@ namespace Math {
       * For example, the absolute value of -5 is the same as the absolute value of 5.
       * @param x A numeric expression for which the absolute value is needed.
       */
-    //% help=math/abs
+    //% help=./reference/math/abs
```

Comment:
(sidenote, I don't think we would typically wan the . here, some weird stuff ends up happening at times with the pathing~)